### PR TITLE
UCT/API: Fix uct_ep_pending_add documentation

### PR DESCRIPTION
**Pull request by dmitrygx**
## What

Fix `uct_ep_pending_add` documentation

## Why ?

Improves UCT API documentation

## How ?

Fix `@param flags` descriprtion

### CODE REVIEW

File: src/uct/api/uct.h [hunk 1]
```diff
@@ -2556,7 +2556,8 @@ UCT_INLINE_API ucs_status_t uct_ep_atomic64_fetch(uct_ep_h ep, uct_atomic_op_t o
  *                    the "func" field.
  *                    After passed to the function, the request is owned by UCT,
  *                    until the callback is called and returns UCS_OK.
- * @param [in]  flags Reserved for future use.
+ * @param [in]  flags Flags that control pending request processing (currently only
```
Comment (yosefe) [inline src/uct/api/uct.h]: can you pls remote "(currently only ...)", replace by `see @ref <name-of-enum>`
even one flag counts..

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12268/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 13-Sep-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/edaa830d5d623ec04150d455294c60f0e70c993d/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/818f7d894a40cefa9d113efb5a5984995590b71f/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/84cd2d46f3af308f50bcb8d875b7008a33458c98/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/1d13b02d876b1f4e1d443809e49a00210d3a7325/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/482862f3d2da7f79921aa7be192e93716162eae1/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/41d9f16e5f08da9bfa12a0587ecdd50ec92faa5e/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/aa265f0b4b0cd1b5d10733ebcd10720185b6d490/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/71c1a5e5c00efa7bcfeba3578d2988c94cf63f3a/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/c8c30dc684fa78910fce7dbbaf793807708dbf4f/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/78126a2e08f01a68b0c1e90dad66aadf653edd23/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/1261a0fbca9893d5816fe6f21bd4cb69fe52b843/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/d7e5a6484a214bd1d12de76b4302fcf598e04cf3/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/ae2fd49c4c3064a09cb41bf79958010e96a56d3b/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/83213ac45c7e78990a7cf72e55fe0095919ff66e/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/8bed9efa23fe16f0b99482ce243d187165fc76ee/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/fc7b143257dc78a43ed8baebbdf1e1a0d6039462/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/e645aa0e2bca4d67230b0d29c579ab99f5c412a7/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/8def142295086520abc1c0048458e1ae92d1bbb6/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/6aa304c0e5cee4c7b1fb3a255acc4b86c32b2ee3/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/f4bf64755a1900a8accf46ec3db8d07717815ee6/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/ca9a92d58cfe16cdd2cdf7fb7d57e9fca6805478/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/01e52f568699081b4871764259ec20979200a9ba/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/205dff1ad67d09cc371ab6913de73fb3bb7e8535/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/1c2766a537bc20fac57e67c955893cdf94aad3ef/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/f258fb8640096625ffa8b0b0805fbcaf30dc8078/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/d914eec877ca979a186be81973bcc1687e3b3dc5/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/737860107b9f6274f5ca5df2105ffaa311e3988e/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/0e129fbb2e0027d435b7c428dc57c90180203fa3/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/244a389eedda9ed20f08e65c70dab4f5/raw/676443105c8fb44c920a174fb8c886c2b7aaaf71/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by shamisp** [comment]
@tonycurtis can you please take a look. thanks !

**Comment by tonycurtis** [comment]
In src/uct/api/uct.h <https://github.com/openucx/ucx/pull/4169#discussion_r321701623>:
> @@ -2556,7 +2556,8 @@ UCT_INLINE_API ucs_status_t uct_ep_atomic64_fetch(uct_ep_h ep, uct_atomic_op_t o
  *                    the "func" field.
  *                    After passed to the function, the request is owned by UCT,

After being passed…
Tony

File: src/uct/api/uct.h (hunk 1 above)
Comment (dmitrygx) [reply src/uct/api/uct.h]: done

**Comment by dmitrygx** [comment]
> In src/uct/api/uct.h <[#4169 (comment)](https://github.com/openucx/ucx/pull/4169#discussion_r321701623)>:
> @@ -2556,7 +2556,8 @@ UCT_INLINE_API ucs_status_t uct_ep_atomic64_fetch(uct_ep_h ep, uct_atomic_op_t o
> * the "func" field. * After passed to the function, the request is owned by UCT, After being passed… Tony

@tonycurtis 👍 done

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12282/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 15-Sep-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/7280e291daf19db7bdc24f66e9a5c0017975cf9f/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/2d866ca8c267ffc0a7f7dd1c34c7dd106b8db666/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/336fea607141db90b7a0829deed2a542a1549638/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/57599ddf3650ce81cd0a87edbd77ae5ce94a5e1c/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/b448acbde93acb6bd9a6886f28a5cbe480fe0339/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/597e44a738762942330c161e10861e1c43ac3653/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/6cecd93b4e40d457a3bf78e1a205918fcbdf3016/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/a956d1406c5314fedffaa7681c7e0b8718f77d4c/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/9b85ae34cb0a5315e2e6888810d0fd6d23b70449/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/9d747709d887033fdd1ad7bc1c9ebf5edd02b8c2/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/242c18d16149e76bc8a93513645a377247ae3d4a/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/c715161e268f33d946b1dfc5bb1c3f80afafe7db/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/12a14f4ad711cbb26d02161d84b681cc25d4ad5e/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/0eaca58b61fd02998d9916f883fd3161815f6b9b/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/7f0f0137f609b5427b7887e5dfbc2b8703aaa4bd/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/96c47913323b07c1c05e56d05920a81cad202f44/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/5a307b53dfcfd6b2480feaa743903f22f2513858/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/32837fb354cacf80815b28171ef0fba44eb58205/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/0454017d2a7f9bdce548cba0a2c14355407c2754/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/63bc063b776ea5c98c9aa6dd2bef1fe19ddf186d/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/f80cd6ce547bb072c1795efc0c0ac287ea99fe46/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/1729e0a2b79ac552d2236338b9dad8e1531cb963/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/fc3b4f7f98113965148f706e47fd5e46912e650f/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/ca84557c7617fab1fa45ecaebbd01affa48bf9f8/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/25e8c61e432ec0aca80dc3d472a5c61f52182617/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/2578223452d0a893b13116586733553acfa97019/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/45164519bef84dc18924c27565e4af9c3e4af0d6/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/aa9f75f8c82e81af2d4643f076316692afa50a01/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/b58c11cc6825d37a8c0ecf1eac5e2708/raw/d4e3ab0ab2526dda8dcb57bb0510cabd378afc70/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**